overlay-debs/mesa-snapshot/mesa-snapshot.sh

```diff
@@ -8,26 +8,26 @@ COMMIT=${COMMIT:-origin/main}
 set -e
 
 DEB_FILE=${DSC_FILE%%.dsc}.debian.tar.xz
-ORIG_FILE=$(echo ${DSC_FILE} | sed -e 's/-.*/.orig.tar.xz/')
+ORIG_FILE=$(echo "${DSC_FILE}" | sed -e 's/-.*/.orig.tar.xz/')
 
 rm -rf mesa
 git clone --depth 1 https://gitlab.freedesktop.org/mesa/mesa
 
 cd mesa
-git fetch --depth 1 origin ${COMMIT##origin/}
+git fetch --depth 1 origin "${COMMIT##origin/}"
```

Comment:
We don't need to quote all the variables. Only the variables that might have whitespaces or might be empty need to be quoted.

Reply:
Yup, shellcheck doesn't know that, so it asks to quote everything by default, but we can add an override to tell it to ignore this particular line; I think it's cheaper to add the double quotes personally, even if it's a bit more verbose